.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,3 +5,15 @@
 
 **Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
 Fixes #
+
+#### Special notes for your reviewer:
+
+#### Does this PR introduce a user-facing change?
+<!--
+If no, just write "NONE" in the release-note block below.
```

Comment:
Can we have release NONE by default and ask the contributor to change it only if necessary (default to no-op)